syserol/COVID.py

```diff
@@ -47,6 +47,8 @@ def Tensor4D():
 
     # Create Tensor 4
     tensor = np.stack(tensors, axis=3)
+    tensor = np.clip(tensor, a_min=1.0, a_max=np.inf)
```

Comment:
I haven't yet tried to normalize the COVID data. I figure we'll rearrange the dimensions first.